doc-surrealdb_versioned_docs/version-latest/querying/surrealql/surrealist.mdx

```diff
@@ -0,0 +1,55 @@
+---
+sidebar_position: 1
+sidebar_label: SurrealQL via Surrealist 
+title: SurrealQL via Surrealist | SurrealQL
+description: In this section, you will explore SurrealQL queries using Surrealist, the official query editor for SurrealDB. Surrealist is a powerful tool that allows you to write, execute, and visualize SurrealQL queries in real-time.
+---
+
+import ThemedImage from '@theme/ThemedImage';
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+# SurrealQL via Surrealist 
+
+In this section, you will explore writing SurrealQL queries using [Surrealist](https://surrealist.app/query), the official query editor for SurrealDB. Surrealist is a powerful tool that allows you to write, execute, and visualize SurrealQL queries in real-time.
+
+The Surrealist query editor provides a rich set of features, including syntax highlighting, query validation, and query execution. You can also view the query results in a tabular format, making it easy to analyze and visualize the data.
```

Comment:
Ah, tabular format! That's the expression I was looking for above when I suggested changing the wording from "immediately" to stressing the nice layout that Surrealist has.